client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -62,6 +64,30 @@ class SiteSettingsFormDiscussion extends Component {
 		);
 	}
 
+	commentDisplaySettings() {
+		const { isJetpack, isCommentsModuleActive, jetpackVersionSupportsCalypsoSettingsUI } = this.props;
```

Comment:
@ryelle I think that's a good idea, but only necessary for JP sites. In other words, make it look like the JP version in this mockup, but only on JP sites.

![image](https://cloud.githubusercontent.com/assets/1123119/22226266/1489bf96-e183-11e6-801d-e951b5b1092e.png)